.circleci/config.yml

```
@@ -2,6 +2,41 @@ version: 2.1
 
 jobs:
 
+  run-formatter:
+    docker:
+      - image: checkstyle/idea-docker:jdk11-idea2023.3.4
+
+    steps:
+      - checkout
+      - run:
+          name: Print versions
+          command: |
+            echo "Maven version:"
+            mvn --version
+            echo "Java version:"
+            java --version
+            echo "IDEA version:"
+            echo $IDEA_VERSION
```

```suggestion
            echo "Maven version:" & mvn --version
            echo "Java version:" & java --version
            echo "IDEA version:" & echo $IDEA_VERSION
```
its one or two then its should turn out the same. But this might be not better.